client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/useInvestigatedPersonInfo.ts

```diff
@@ -130,15 +147,18 @@ const useInvestigatedPersonInfo = (): InvestigatedPersonInfoOutcome => {
     const handleCannotCompleteInvestigationCheck = (cannotCompleteInvestigation: boolean) => {
         setInvestigationStatus({
```

Comment:
There is no need in checking the same condition every time - do it one time and created two different objects

Reply:
unused function, deleted